app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,122 @@
+<resources>
+    <string name="app_name" translatable="false">Sysctl GUI</string>
+    <string name="splash_status_checking_root">Kök erişimi denetleniyor</string>
+    <string name="splash_status_checking_busybox">Busybox kontrol ediliyor</string>
+    <string name="splash_status_checking_migration">Eski veritabanı şeması kontrol ediliyor</string>
+    <string name="splash_status_performing_migration">Veritabanı taşıma gerçekleştiriliyor</string>
+    <string name="error">Hata</string>
+    <string name="settings">Ayarlar</string>
+    <string name="about">Hakkında</string>
+    <string name="show_variables">Değişkenleri göster</string>
+    <string name="show_variables_sum">Tüm çekirdek parametrelerini bir listede görüntüleme ve düzenleme</string>
+    <string name="sysctl_operations">Sysctl işlemleri</string>
+    <string name="read_from_file">Dosyadan değerleri oku</string>
+    <string name="read_from_file_sum">Seçilen bir .conf veya .json dosyasından parametre yüklemek için bu seçeneği kullanın</string>
+    <string name="show_favorites">Favorileri göster</string>
+    <string name="show_favorites_sum">Bu listeden en sevdiğiniz çekirdek parametrelerine erişin</string>
+    <string name="exit">Çıkış</string>
+    <string name="kernel_params">Çekirdek parametreleri</string>
+    <string name="browse_variables">Değişkenlere göz at</string>
+    <string name="browse_variables_sum">Belirli bir çekirdek parametresi için /proc/sys'ye göz atın</string>
```

Comment:
This string resource contains an unescaped apostrophe (`'`)

Same for lines [92 and 93](https://github.com/Lennoard/SysctlGUI/pull/62/files#diff-11548b2158b255e354ac7ab1cdc4dc4bf1d0cb372bbf1425508d86d35b69ed50R92-R93)

Suggestion: fix apostrophe escaping by using a `\` character before each.